VKSample/VKSample/Modules/FriendsModule/Cells/FriendTableViewCell.swift

```diff
@@ -5,15 +5,41 @@ import UIKit
 
 /// User-friend cell.
 final class FriendTableViewCell: UITableViewCell {
+    // MARK: - Private Constants.
+
+    private enum Constants {
+        static let userPlaceholderString = "userPlaceholder"
+    }
+
     // MARK: - Private @IBOutlet.
 
     @IBOutlet private var friendImageView: UIImageView!
     @IBOutlet private var nameLabel: UILabel!
 
     // MARK: - Public methods.
 
-    func configure(with friend: Friend) {
-        friendImageView.image = UIImage(named: friend.avatarImageName)
-        nameLabel.text = "\(friend.name.firstName) \(friend.name.lastName ?? "")"
+    func configure(with friend: ItemFriend) {
+        guard let avatar = friend.avatar else { return }
+        setImage(avatarURLText: avatar)
+        nameLabel.text = "\(friend.firstName) \(friend.lastName)"
+    }
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        friendImageView.image = UIImage(named: Constants.userPlaceholderString)
+    }
+
+    // MARK: - Private methods.
+
+    private func setImage(avatarURLText: String) {
```

Comment:
```suggestion
    private func setImage(avatarURLText: String) {
```
у тебя в каждом классе ячейки этот метод, напиши расширение, которое загружает картинку и юзай его

Reply:
плюсую